xml/operations-maintenance-controller-pit_database_recovery.xml

```diff
@@ -254,24 +270,24 @@ hostname = {{ hostname of the first &mariadb; node }}</screen>
    </listitem>
    <listitem>
     <para>
-     From the &lcm;, execute the following playbook to start all
-     &mariadb; instances:
+     From the &lcm;, execute the following playbook to start all &mariadb;
+     instances:
     </para>
 <screen>cd ~/scratch/ansible/next/ardana/ansible
 ansible-playbook -i hosts/verb_hosts percona-start.yml</screen>
    </listitem>
    <listitem>
     <para>
-     &mariadb; cluster status can be checked using the <literal>percona-status.yml
-     </literal>playbook:
+     &mariadb; cluster status can be checked using the
+     <literal>percona-status.yml </literal>playbook:
     </para>
 <screen>cd ~/scratch/ansible/next/ardana/ansible
 ansible-playbook -i hosts/verb_hosts percona-status.yml</screen>
    </listitem>
    <listitem>
     <para>
-     On all of the nodes running the &mariadb; service, run the following commands
-     as root:
+     On all of the nodes running the &mariadb; service, run the following
+     commands as root:
     </para>
 <screen>sudo su
 touch /var/lib/mysql/percona.initialised
```

Comment:
Not "run as root, nor sudo su:

          <para>
           On all of the nodes running the &mariadb; service, run the following commands:
          </para>
          <screen>&prompt.sudo;touch /var/lib/mysql/percona.initialised
          &prompt.sudo;chown mysql:mysql /var/lib/mysql/percona.initialised</screen>